data/human/marauders.txt

```diff
@@ -248,7 +254,7 @@ ship "Marauder Falcon"
 		"engine capacity" 165
 		"reverse thruster slot" 1
 		"steering slot" 1
-		"thruster slot" 1
+		"thruster slot" 2
```

Comment:
```suggestion
		"thruster slot" 1
```

This one probably shouldn't be increased. The description doesn't mention maneuverability or engines at all.